### DISCUSSION TITLE
Hydration completed but contains mismatches.

### DISCUSSION THREAD

**Discussion by thegears** [nuxt/nuxt#16277] [Questions]
### Environment

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.0`
- Nuxt Version:     `0.0.0`
- Nitro Version:    `-`
- Package Manager:  `unknown`
- Builder:          `webpack`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

https://github.com/nuxt/framework

### Describe the bug

My code:

```
<template>
	<div  v-if="!loading">
		<div>
			<NuxtPage :user="user"/>
		</div>
	</div>
</template>
```

Error:
![image](https://user-images.githubusercontent.com/65253167/201522420-29439acc-58ec-4482-9366-bebdf6eec010.png)

**Comment by TheAlexLichter** [comment]
Hydration errors can have [multiple causes](https://blog.lichter.io/posts/vue-hydration-error/) like invalid HTML. It all comes down to having different state on the server and on the client.

Please provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction) for more insights.

**Comment by spelling203** [comment]
Hello, are you done with it
I have the same problem

**Comment by motea927** [comment]
I think your loading state logic may have some error,
So when server-side rendering loading is **true**,
And when client-side rendering loading is **false**,
How do you change loading state? Maybe API Call's method is wrong.
if you can provide [minimal reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction), we can figure out the problem soon.
